src/main/archetype/ui.frontend/src/main/webpack/components/content/languagenavigation/scss/languagenavigation.scss

```diff
@@ -0,0 +1,43 @@
+/* BEM Description
+BLOCK cmp-languagenavigation
+    ELEMENT cmp-languagenavigation__group
+    ELEMENT cmp-languagenavigation__item
+        MOD cmp-languagenavigation__item--active
+        MOD cmp-languagenavigation__item--countrycode-*
+        MOD cmp-languagenavigation__item--langcode-*
+        MOD cmp-languagenavigation__item--level-*
+    ELEMENT cmp-languagenavigation__item-link
+    ELEMENT cmp-languagenavigation__item-title
+*/
+
+.cmp-languagenavigation {
+  &__group {
+
+  }
+
+  &__item {
+    &--active {
+
+    }
+
+    //&--countrycode-en {
+    //
+    //}
+
+    //&--langcode-en {
```

Comment:
We could provide different examples here for countrycode and langcode to show the difference:

```css
// .cmp-languagenavigation--countrycode-US
// .cmp-languagenavigation--langcode-en-US
```